script/data-loader/src/lib.rs

```diff
@@ -0,0 +1,13 @@
+use ckb_core::cell::CellMeta;
+use ckb_core::extras::BlockExt;
+use ckb_core::transaction::CellOutput;
+use numext_fixed_hash::H256;
+
+/// Script DataLoader
+/// abstract the data access layer
+pub trait DataLoader {
+    // load CellOutput
+    fn lazy_load_cell_output(&self, _cell: &CellMeta) -> CellOutput;
```

Comment:
Why use underscore before parameter name?

Reply:
If there is no default implementation, no need underscore.